web/src/beta/features/Editor/useLeftPanel.tsx

```diff
@@ -73,32 +75,46 @@ export default ({
     switch (tab) {
       case "map":
         return (
-          <MapSidePanel
-            scene={scene}
-            layers={nlsLayers}
-            selectedLayerId={selectedLayerId}
-            selectedSceneSetting={selectedSceneSetting}
-            onLayerDelete={onLayerDelete}
-            onLayerNameUpdate={onLayerNameUpdate}
-            onLayerVisibilityUpate={onLayerVisibilityUpate}
-            onLayerSelect={onLayerSelect}
-            onSceneSettingSelect={onSceneSettingSelect}
-            onDataSourceManagerOpen={onDataSourceManagerOpen}
-            onFlyTo={onFlyTo}
-          />
+          <Resizable
```

Comment:
Hmm can you find a way not to have to duplicate Resizable this much? If we add more tabs in the future super quick this file would get much bigger than necessary.
Some options might be:
- Move in to SidePanel which is used by MapSidePanel and StorySidePanel as SidePanelCommon
- do a bit more in this leftPanel memo
- Move back to where it was (for now)
- etc

Reply:
@KaWaite  Think we need to discuss this